### ISSUE TITLE
Wiki is Broken

### ISSUE THREAD

**Issue by Ugdhar** [modernuo/ModernUO#1137]
It says "head on over to getting started", but just clicking on getting started brings you to a 404 page (clicking the link that says head on over") or brings you back to the same welcome wiki page (clicking getting started in the ToC)

**Comment by kamronbatman** [comment]
Yeah I switched to MKDocs and hosted it on modernuo.com. As soon as I get some time I'll update that.

**Comment by Valdur5** [comment]
Would be nice, I got it running but I can't find any doc about which commands are available for example.

**Comment by kamronbatman** [comment]
Yeah, RunUO had a docs generator and a website set up for this. I had forums set up but no time to manage it and nobody used it so I took it down. I would love help in filling out the wiki.

**Comment by DerekGooding** [comment]
Just a little late to the party.

@Ugdhar 
I'm on it. Plans to completely flesh out the wiki and to ask Discord for anything they'd like to see added.
